mockDB.js

```diff
@@ -0,0 +1,103 @@
+var config = require('./config.js'),
+    crudService = require('./scripts/crud-service.js'),
+    doc = require('./scripts/doc-config.js'),
+    usage = require('./scripts/usage.js');
+
+var url = config.db.url,
+    cmds = [
```

Comment:
Use a map! This is an O(N) implementation when it can be O(1)!  Also, put this in a separate config file if you need to access in other parts of your app.